webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/ncf/TechniqueWriter.scala

```diff
@@ -761,11 +765,11 @@ class ClassicTechniqueWriter(basePath : String, parameterTypeService: ParameterT
       def bundleMethodCall( parentBlocks : List[MethodBlock])(method : MethodElem) : List[(String, String)] = {
         method match {
           case c : MethodCall =>
-            val call = MethodCall.renameParams(c,methods)
+            val call = MethodCall.renameParams(c,methods).copy(methodId = BundleName("log_na_rudder"))
             (for {
-              method_info <- methods.get(call.methodId)
+              method_info <- methods.get(c.methodId)
```

Comment:
it the parameter is not renamed, there's a possibility that the methods.get fails

Reply:
ok, it is called like that everywhere else on the file, so it should be ok